src/main/java/jp/co/soramitsu/sora/sdk/did/model/serializer/TimeCombinedSerializer.java

```diff
@@ -0,0 +1,43 @@
+package jp.co.soramitsu.sora.sdk.did.model.serializer;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import java.io.IOException;
+import java.text.ParseException;
+import jp.co.soramitsu.sora.sdk.did.model.dto.DateTimeParseException;
+import jp.co.soramitsu.sora.sdk.did.validation.ISO8601DateTimeFormatter;
+
+public class TimeCombinedSerializer {
```

Comment:
Why do we need to create serializer/deserializer if we don't customize returning types?

Reply:
When you create DDO from json, there will be no validation by default (validation that String from JSON, is really ISO8601 formatted time, represented as String).

Since we check input type at builders, it is probably ok to remove serializer.